tests/apis.js

```diff
@@ -1,88 +1,54 @@
 import test from 'ava';
-import uuid from 'uuid';
-import ipsum from 'lorem-ipsum';
-import Promise from 'bluebird';
 import slugify from 'underscore.string/slugify';
 import isEqual from 'lodash/isEqual';
 import cloneDeep from 'lodash/cloneDeep';
+import isUUID from 'validator/lib/isUUID';
+
 import apiUtils from '../lib/api/utils';
 import api from '../lib/api';
-import utils from '../lib/utils';
 import database from '../lib/database';
+import merged from './fixtures/live/objects/types-models-merged';
+import dbmocks from './fixtures/live/objects/database-mocks.js';
 
-const content = [];
-const types = [];
-const eachOfType = {};
-const allTypes = [];
-
-for (let i = 0; i < 5; i++) {
-  types.push(`Test Type ${ipsum({
-    count: 1,
-    units: 'words',
-  })}`);
-  eachOfType[types[i]] = 0;
-  allTypes.push({
-    name: types[i],
-    id: slugify(types[i]),
-    description: ipsum({
-      count: Math.round(Math.random() * 6 + 1),
-      units: 'words',
-    }),
-    attributes: [],
-  });
-}
+const content = dbmocks.rows;
+const types = merged.types;
+const eachOfType = dbmocks.eachOfType;
+const allTypes = merged.allTypes;
 
-for (let i = 0; i < 50; i++) {
-  const name = ipsum({
-    count: 3,
-    units: 'words',
-    format: 'plain',
+test.cb.before(t => {
+  database.init().then(() => {
+    database('live').insert(content).then(() => {
+      t.end();
+    }).catch(e => { // because we can't return in a before, this catch doesn't bubble out
```

Comment:
Can you put this on the next line instead of inline with the function initilization